collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/MyanmarDatePickerDialog.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.fragments.dialogs;
+
+import org.javarosa.core.model.FormIndex;
+import org.joda.time.LocalDateTime;
+import org.odk.collect.android.logic.DatePickerDetails;
+import org.odk.collect.android.utilities.MyanmarDateUtils;
+
+import mmcalendar.MyanmarDate;
+import mmcalendar.MyanmarDateKernel;
+
+public class MyanmarDatePickerDialog extends CustomDatePickerDialog {
+    private static final int MIN_SUPPORTED_YEAR = 1261; //1900 in Gregorian calendar
+    private static final int MAX_SUPPORTED_YEAR = 1462; //2100 in Gregorian calendar
```

Comment:
This is less than 100 years from now, is there a reason this is the max? Any docs you can link to?

Reply:
It's because we use the same range in other calendars I mean 1900-2100 we can improve it later.